packages/libs/react-components/src/styles/stitches.config.ts

```diff
@@ -4,8 +4,10 @@ import { colors, colorsDark } from './colors';
 
 import type { CSS as StitchesCSS } from '@stitches/react';
 import { createStitches, PropertyValue } from '@stitches/react';
+import { ConfigType } from '@stitches/react/types/config';
 
-const sizes: Record<string, string> = {
+// eslint-disable-next-line @kadena-dev/typedef-var
```

Comment:
Should we add the type back here too? I think it would be better to use eslint-disable as little as possible

Reply:
yes